voltron/plugins/view/memory.py

```diff
@@ -85,13 +88,21 @@ def render(self, results):
                     chunk = m_res.memory[c:c + self.args.bytes]
                     addr_str = self.colour(self.format_address(m_res.address + c, size=target['addr_size'], pad=False),
                                            self.config.format.addr_colour)
+
+                    if self.args.track and self.last_memory and self.last_address == m_res.address:
+                        bytes = []
+                        for i, x in enumerate(six.iterbytes(chunk)):
+                            if x != self.last_memory[c + i]:
+                                bytes.append(self.colour("%02X" % x, 'red'))
+                            else:
+                                bytes.append("%02X" % x)
+                        memory_str = ' '.join(bytes)
+                    else:
+                        memory_str = ' '.join(["%02X" % x for x in six.iterbytes(chunk)])
+
                     if self.args.deref:
-                        info_str = ''
-                        if len(chunk) == target['addr_size']:
```

Comment:
I think this check was redundant: `len(chunk)` comes from the slice above, whose length is `self.args.bytes` (see [here](https://github.com/snare/voltron/pull/168/files#diff-666f65600001e7d8fada866226fbee57R88)), but `self.args.bytes` is already set to match `target['addr_size']` [here](https://github.com/snare/voltron/pull/168/files#diff-666f65600001e7d8fada866226fbee57R83)